test/schemas/store-order-recipient.test.js

```diff
@@ -61,13 +61,13 @@ describe('isNumberOrUuid', () => {
     ${[1, 2, 3]}                | ${false}
     ${null}                     | ${false}
     ${undefined}                | ${false}
-    ${-100}                     | ${false}
-    ${''}                       | ${false}
-    ${'fakeUuid'}               | ${false}
+    ${-100}                     | ${true}
+    ${''}                       | ${true}
+    ${'fakeUuid'}               | ${true}
```

Comment:
We should either add another condition here, or change `'fakeUuid'` to `faker.random.alphaNumeric()` (or whatever)